documentation/starter-kits/README.md

```diff
@@ -46,21 +46,31 @@ To leverage this template, make sure to do the following:
 
 A contribution guide is an important document directed at *new contributors* to your project - to help them in understanding what and how they may contribute. It can be used to outline expected requirements prior to contributions (i.e. acceptance of licensing terms, signing off on any developer agreements required) as well as to ensure the quality of contributions are in-line with expected project norms. If a developer asks how to contribute, this is the guide to point them to, and if patches are submitted to a project but are missing essential ingredients, this is again the guide to point new developers to.
 
-We provide a collections of sample contribution guide templates below, but please keep in mind to modify and adapt to your project as needed.
+We provide a sample contribution guide template below, but please keep in mind to:
+- Modify and adapt to your project as needed. You may not need all features of the template guide. 
+- Replace all instances of `[INSERT ...]` within the template with your project's specific environment.
 
-### Limited Contributions Template
+### Contributing Guide Template
 
-This template is meant for projects that only accept bug-reports and issues, not necessarily code patches or other diverse artifacts. 
+This template covers the following contributions areas:
+- Prerequisites to contributing
+- Development process
+- Ways to contribute, including: issues, code, documentation, media, questions, etc.
```

Comment:
This template covers the following areas: 
...
- Ways to contribute issues, code, documentation, media and questions, among other topics.